src/com/google/javascript/rhino/JSDocInfo.java

```diff
@@ -2042,13 +2042,17 @@ public String getFileOverview() {
 
       if (info.extendedInterfaces != null) {
         for (JSTypeExpression interfaceType : info.extendedInterfaces) {
```

Comment:
I think I'd rather go back and make sure that info.extendedInterfaces never contains nulls. This is probably fine as a short-term fix though, if that turns out to be too difficult.